### DISCUSSION TITLE
Alt + p won't function anymore

### DISCUSSION THREAD

**Discussion by ttx99** [pystardust/ytfzf#639] [Q&A] [answered]
I cant scrape next page how to fix? version 2.5.4

**Comment by Euro20179** [comment, marked as answer]
use `ctrl-p`

**Reply by ttx99** [reply]
pls revert back the old functionality next version cause im pressing ctrl-p and its not scraping anything the search result numbers still the same (79/79) installed using a package manager.
![Screenshot_2023-02-05_14-15-44](https://user-images.githubusercontent.com/95531944/216804780-6bc6fe27-deb0-4c37-a580-00f30f7404df.png)

**Reply by Euro20179** [reply]
Oops, it does seem to have broken. Pushed a fix to the development branch.
